### ISSUE TITLE
Something about the result?

### ISSUE THREAD

**Issue by Lr-2002** [Snagnar/Hieros#2]
Sorry to bother again.
I've trained the model using almost the same config as you, While the result seems not very good, May you help me or give me some suggestion?
Firstly, I'm using the batch_size = 8 (because of the 4090 has only 24G )while you are using 16 
This is the trained image in tensorflow after 153k steps

https://github.com/Snagnar/Hieros/assets/42893558/a83e16cc-1c93-4c66-87f5-e00946da4b00

https://github.com/Snagnar/Hieros/assets/42893558/38e51809-47fb-4ac3-b020-8b3a56c5464a

It seems the policy has no move ?

**Comment by Snagnar** [comment]
Unfortunately Hieros is highly sensitive to changes in the hyperparameters for breakout, as the overalll performance isn't even good with the specific parameters. Reducing the batch size could have such an effect. It is possible for the policy distribution to collapse on one single action for games like pong and breakout. Also the performance can be different when using a different seed.

With using almost the same config as me, do you mean the default parameters defined in the configs.yaml? because unfortunately these are not the same as have been used for the experiments in the paper. I realize now that this could be confusing. You can find the actually used parameters in experiments/atari100k_sweep.yml

I will also change the default configs.yml so that it actually has the default values. Sorry for the confusion.

**Comment by Lr-2002** [comment]
₍₍ ง(*Ӧ)ว ⁾⁾
okk thanks a lot, i'll try your hyperparams latter❤️❤️❤️
